evm/src/cpu/kernel/tests/test_utils.rs

```diff
@@ -0,0 +1,35 @@
+use std::ops::Range;
+
+use anyhow::Result;
+use ethereum_types::U256;
+
+use crate::cpu::kernel::aggregator::KERNEL;
+use crate::cpu::kernel::interpreter::Interpreter;
+use crate::memory::segments::Segment;
+use crate::witness::memory::MemoryAddress;
+
+pub struct InterpreterSetup {
```

Comment:
Is this code duplicated from `kernel/interpreter.rs`? It seems like this one isn't actually imported anywhere?

Reply:
you're right; this was an accidental vestige of a previous iteration of this PR---just deleted it :)